wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/SuccessfulCertificateEnrollmentViewController.swift

```diff
@@ -0,0 +1,181 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireSyncEngine
+
+final class SuccessfulCertificateEnrollmentViewController: UIViewController {
+
+    // MARK: - Properties
+
+    private let titleLabel: UILabel = {
+        let label = DynamicFontLabel(
+            text: L10n.Localizable.EnrollE2eiCertificate.title,
+            style: .bigHeadline,
+            color: SemanticColors.Label.textDefault)
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.accessibilityIdentifier = "titleLabel"
```

Comment:
are accessibility identifier correct here? I thought we localize them

Reply:
Yes, it's correct. `accessibilityIdentifier` is for QA automation. `accessibilityLabel` is for VoiceOver, but it will use the same value as label text